lightkurve/lightcurve.py

```diff
@@ -1233,12 +1235,26 @@ class FoldedLightCurve(LightCurve):
     """
     def __init__(self, *args, **kwargs):
         self.time_original = kwargs.pop("time_original", None)
+        self.period = kwargs.pop("period", None)
+        self.t0 = kwargs.pop("t0", None)
         super(FoldedLightCurve, self).__init__(*args, **kwargs)
 
     @property
     def phase(self):
         return self.time
 
+    @property
+    def odd(self):
```

Comment:
Elsewhere in Lightkurve we have often used the suffix `_mask` to mark boolean masks.  Should we name this `odd_mask` or `odd_cadence_mask`?

Reply:
I'm happy with that suggestion, I kept it short because it is a little easier to read when we have a long "sentence". I'll make this change.